src/main/java/uk/gov/hmcts/darts/cases/mapper/CasesMapper.java

```diff
@@ -43,6 +45,9 @@ public class CasesMapper {
     private final HearingReportingRestrictionsRepository hearingReportingRestrictionsRepository;
     private final CaseRetentionRepository caseRetentionRepository;
     private final AuthorisationApi authorisationApi;
+    private final LogApi logApi;
+    @Value("${darts.log.unallocated-case-regex}")
+    private final String unallocatedCaseRegex;
```

Comment:
Consider:

```suggestion
    private final Pattern unallocatedCaseRegex;
```
This has the advantage of validating the provided regex sytax upon application startup (when the property value is injected and the `CaseMapper` bean is created). If a junk regex is provided the application will fail fast, rather than waiting until the method is invoked at runtime.